examples/ruby-image-upload/Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+gem 'cloudinary'
+group :development do
+  gem 'dotenv-rails'
```

Comment:
curious about this one, it imports in the ruby file as `dotenv`, is that accurate?

Reply:
Yes this will allow us to use env